bin/start_lke.sh

```diff
@@ -346,13 +346,17 @@ header "Bank of Sirius"
 cd "${script_dir}/../pulumi/python/kubernetes/applications/sirius"
 pulumi $pulumi_args up
 
-header "Finished!!"
+THE_FQDN=$(pulumi config get kic-helm:fqdn -C ${script_dir}/../pulumi/python/config)
+THE_IP=$(kubectl  get service kic_nginx_ingress  --namespace nginx-ingress --output=jsonpath='{.status.loadBalancer.ingress[*].ip}')
+
+header "Finished!"
 echo " "
 echo "The startup process has finished successfully"
 echo " "
+echo " "
 echo "Next Steps:"
 echo " "
-echo "1. Map the IP address of your Ingress Controller with your FQDN."
+echo "1. Map the IP address ($THE_IP) of your Ingress Controller with your FQDN ($THE_FQDN)."
```

Comment:
Nit: I feel like we run the risk of having this nice messaging diverge with it spread around all over the place. However I know you said that these bash scripts may not be long for this world so maybe it's not worth the effort to centralize it.

Reply:
You're not wrong; I made a note to bring this up in our next tech meeting to see what thoughts are around this.